build.gradle

```diff
@@ -9,7 +9,7 @@
 buildscript {
     ext.android_support_version = '28.0.0'
     ext.kotlin_version = '1.3.0'
-    ext.android_components_version = '7.0.0'
+    ext.android_components_version = '9.0.0-SNAPSHOT'
```

Comment:
Obv change this to `9.0.0` when it's released

Reply:
👍